src/dc/nipals.js

```diff
@@ -18,22 +18,15 @@ export default class nipals {
       } else {
         Y = WrapperMatrix2D.checkMatrix(Y);
       }
-      if (!Y.isColumnVector() || Y.rows !== X.rows) {
-        throw new Error('Y must be a column vector of length X.rows');
```

Comment:
adapt the text of the error message as well to reflect the results of the new test